shared/dtos.d.ts

```diff
@@ -6,6 +6,7 @@ interface Item {
 interface Pizza extends Item {
     description: string;
     price: { [version: string]: number };
+    tags: string[];
```

Comment:
`Item` should have tags, `Pizza` will then inherit tags from `Item`

`BasketItem` should NOT have tags (this will enforce us using it from the catalog, not from the user's request)